libs/javalib/src/mill/javalib/JavaModule.scala

```diff
@@ -964,14 +971,19 @@ trait JavaModule
         bspLocalRunClasspath(needsToMergeResourcesIntoCompileDest)()
     }
 
+  /**
+   * [[compileClasspathFor]] for regular compilations.
+   *
+   * Keep return value in sync with [[bspCompileClasspath]].
+   */
+  def compileClasspath: T[Seq[PathRef]] = Task { compileClasspathFor(CompileFor.Regular)() }
```

Comment:
Does this need to be public?

Reply:
It was previously public, so changing the visibility would break binary compatibility.

Reply:
Looks like I commented on the wrong method.